creator-node/src/routes/audiusUsers.js

```diff
@@ -66,26 +97,32 @@ module.exports = function (app) {
       return errorResponseBadRequest(e.message)
     }
 
-    const t = await models.sequelize.transaction()
+    const transaction = await models.sequelize.transaction()
+
     try {
-      // Insert / update audiusUser entry on db.
-      const audiusUser = await models.AudiusUser.upsert({
+      logger.info(`beginning audiusUsers DB transactions`)
+
+      await updateClockInCNodeUserAndClockRecords(req, 'AudiusUser', transaction)
+
+      // Insert new audiusUser entry to DB
+      const audiusUser = await models.AudiusUser.create({
         cnodeUserUUID,
         metadataFileUUID,
         metadataJSON,
         blockchainId: blockchainUserId,
         coverArtFileUUID,
-        profilePicFileUUID
-      }, { transaction: t, returning: true })
+        profilePicFileUUID,
+        clock: models.sequelize.literal(`(${selectCNodeUserClockSubquery(cnodeUserUUID)})`)
```

Comment:
this pattern seems dangerous. we should pass in the object to create as part of the updateClockInCNodeUserAndClockRecords and have clockManager manage adding clock to the record

Reply:
fixed